lib/server/db/postgres/postgres.go

```diff
@@ -120,12 +133,18 @@ func (p *Postgres) Create() (*db.DB, error) {
 
 // CreateDatabase creates database
 func (p *Postgres) CreateDatabase() (*db.DB, error) {
-	dbName := p.dbName
-	err := p.createDatabase()
+	exists, err := exists(p.SqlxDB, p.dbName)
 	if err != nil {
-		return nil, errors.Wrap(err, "Failed to create Postgres database")
+		return nil, err
+	}
+	if !exists {
+		err = p.createDatabase()
```

Comment:
In `createDatabase` we still have the following block of code.

```
	if err != nil {
		if !strings.Contains(err.Error(), fmt.Sprintf("database \"%s\" already exists", dbName)) {
			return errors.Wrap(err, "Failed to execute create database query")
		}
	}
```

Shouldn't that be killed now?

Reply:
Missed this, thank you